charts/datasaur/values.yaml

```diff
@@ -175,4 +175,126 @@ loki:
       logLevel: info
       serverPort: 3101
       clients:
-        - url: http://{{ .Release.Name }}-loki:3100/loki/api/v1/push
\ No newline at end of file
+        - url: http://{{ .Release.Name }}-loki:3100/loki/api/v1/push
+
+analytics:
+  enabled: false
+
+metabasemysql:
+  primary:
+    resources:
+      requests:
+        cpu: 100m
+        memory: 1500Mi
+
+    persistence:
+      size: 1Gi
+
+    configuration: |-
+      [mysqld]
+      skip-log-bin
+      default_authentication_plugin=mysql_native_password
+      skip-name-resolve
+      explicit_defaults_for_timestamp
+      basedir=/opt/bitnami/mysql
+      plugin_dir=/opt/bitnami/mysql/lib/plugin
+      port=3306
+      socket=/opt/bitnami/mysql/tmp/mysql.sock
+      datadir=/bitnami/mysql/data
+      tmpdir=/opt/bitnami/mysql/tmp
+      max_allowed_packet=16M
+      bind-address=0.0.0.0
+      pid-file=/opt/bitnami/mysql/tmp/mysqld.pid
+      log-error=/opt/bitnami/mysql/logs/mysqld.log
+      character-set-server=UTF8
+      collation-server=utf8_general_ci
+      [client]
+      port=3306
+      socket=/opt/bitnami/mysql/tmp/mysql.sock
+      default-character-set=UTF8
+      plugin_dir=/opt/bitnami/mysql/lib/plugin
+      [manager]
+      port=3306
+      socket=/opt/bitnami/mysql/tmp/mysql.sock
+      pid-file=/opt/bitnami/mysql/tmp/mysqld.pid<root_password>
+      expire_logs_days=3
+  auth:
+    password: admin
+    rootPassword: admin
+    createDatabase: true
+    database: metabase
+
+metabase:
+  image:
+    repository: public.ecr.aws/glair/metabase
+    tag: v0.44.3
+  resource:
+    request:
+      cpu: 100m
+      memory: 800Mi
+  database:
+    type: mysql
+    connectionURI: "mysql://root:{{ .Values.metabasemysql.auth.rootPassword }}@{{ .Release.Name }}-metabasemysql:3306"
+  replicaCount: 1
+  clearcache:
+    enabled: true
+    schedule: "0 */3 * * *"
+    secret:
+      db_host: null
+      db_name: metabase
+      db_user: root
+      db_pass: admin
+
+metabaseImporter:
+  enabled: true
+  secret:
+    aws:
+      access_key: null
+      secret_key: null
+      region: null
+      s3_bucket: null
+      athena_workgroup: null
+      glue_catalog_name: null
+  resources:
+    limits:
+      cpu: 100m
+      memory: 1500Mi
+    requests:
+      cpu: 10m
+      memory: 800Mi
+
+glairAgent:
+  config:
+    s3_bucket: null
+    cron_expression: "0 */3 * * *"
+    run_backfill: true
+
+  image:
+    repository: public.ecr.aws/glair/glair-agent
+    tag: v0.1.20
```

Comment:
nanti tanggung jawab untuk naikin version glair-agent dan metabase siapa?

apakah datasaur atau customer?

Reply:
bump @daniel-jeremia @billy-editiano

Reply:
kalo self-hosted jadinya client nih, sekalian sama datasaur version upgrade kali ya @tonoman3g

Reply:
oke berarti ini as it is ya, jadi user ga perlu override `image.tag: v0.1.20` ya @daniel-jeremia @billy-editiano